src/main/java/org/broadinstitute/hellbender/engine/spark/SparkCommandLineArgumentCollection.java

```diff
@@ -17,17 +19,29 @@
 public final class SparkCommandLineArgumentCollection implements Serializable {
     private static final long serialVersionUID = 1L;
 
-    @Argument(fullName = "spark-master", doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.", optional = true)
+    public static final String SPARK_MASTER_LONG_NAME = "spark-master";
+    public static final String SPARK_VERBOSITY_LONG_NAME = "spark-verbosity";
+
+    @Argument(
+            doc="URL of the Spark Master to submit jobs to when using the Spark pipeline runner.",
+            fullName = SPARK_MASTER_LONG_NAME,
+            optional = true)
     private String sparkMaster = SparkContextFactory.DEFAULT_SPARK_MASTER;
 
     @Argument(
-            doc = "spark properties to set on the spark context in the format <property>=<value>",
-            shortName = StandardArgumentDefinitions.SPARK_PROPERTY_NAME,
+            doc = "Spark properties to set on the Spark context in the format <property>=<value>",
             fullName = StandardArgumentDefinitions.SPARK_PROPERTY_NAME,
+            shortName = StandardArgumentDefinitions.SPARK_PROPERTY_NAME,
```

Comment:
Don't need to specify shortName and longName if they are the same, just specify long name.

Reply:
Done